src/test/skript/environments/extra/paper-1.17.json

```diff
@@ -4,7 +4,7 @@
 		{"source": "server.properties.generic", "target": "server.properties"}
 	],
 	"downloads": [
-		{"source": "https://papermc.io/api/v1/paper/1.17/latest/download", "target": "paperclip.jar"}
+		{"source": "https://papermc.io/api/v2/projects/paper/versions/1.17/builds/79/downloads/paper-1.17-79.jar", "target": "paperclip.jar"}
```

Comment:
Is it possible to always download the latest of 1.17? (This could apply for other versions in the past, but they are no longer supported so the last build number is already known so it doesn't matter as much)

Reply:
> Edit: nevermind, this is for 1.17 not 1.17.1 lol

lol :D 
I think that's possible to get the latest of the json array list but not sure how to do that, I will give it a try when I can

Reply:
If you could please change this to 1.17.1, I would like to get this merged ASAP (so PR builds actually run)
At a later date, we can update our setup to fetch the latest release, but as of now it would be nice to see PRs actually build.

Reply:
Sure, should be ready within minutes